src/Simulate.js

```diff
@@ -0,0 +1,16 @@
+import { MissionUtils } from '@woowacourse/mission-utils';
+const Random = MissionUtils.Random;
+
+export function processInputToOutput(driverArray, totalPhase) {
+  let result = [];

```

Comment:
```suggestion
  const result = [];
```
`result` 변수는 재할당되지 않고 `push` 메서드를 사용해서 원소를 추가하기 때문에 `const`로 선언해도 무방할 것 같아요~!